api/config.js

```diff
@@ -2,3 +2,8 @@ import path from 'path'
 
 export const distPath = path.resolve(__dirname, 'dist')
 export const apiEndpoint = '/graphql'
+export const rethinkdb = {
```

Comment:
Lets populate these from the `.env` file, and add these suggested values to `.env.example`.